packages/node-core/package.json

```diff
@@ -29,6 +29,7 @@
     "@subql/x-merkle-mountain-range": "^2.0.0-0.1.2",
     "@willsoto/nestjs-prometheus": "^4.4.0",
     "lodash": "^4.17.21",
+    "node-lfu-cache": "5.0.0",
```

Comment:
What made you choose this version? It seems to be no longer maintained and is based off https://github.com/isaacs/node-lru-cache and this one has TS support as well